pkg/preparer/setup.go

```diff
@@ -32,6 +32,7 @@ type PreparerConfig struct {
 	HooksDirectory       string                 `yaml:"hooks_directory"`
 	CAPath               string                 `yaml:"ca_path,omitempty"`
 	PodRoot              string                 `yaml:"pod_root,omitempty"`
+	StatusPort           int                    `yaml:"status_port"`
```

Comment:
In a future change, we should have the preparer write the status port from the root values of the pod manifest to the `PLATFORM_CONFIG_PATH`. As written, this requires the `status_port` field to be repeated as top-level conf and as a value in the config stanza. In general, we need to figure out how to address better port values.